packages/@aws-cdk/aws-lambda/lib/log-retention.ts

```diff
@@ -69,6 +94,7 @@ export class LogRetention extends cdk.Construct {
       properties: {
         ServiceToken: provider.functionArn,
         LogGroupName: props.logGroupName,
+        LogRetentionRetryOptions: props.logRetentionRetryOptions,
```

Comment:
Just `Retry` or `SdkRetry`. We're already in the context of LogRetention.

Reply:
Ok, will change